public/javascript/script.js

```diff
@@ -0,0 +1,90 @@
+const video = document.getElementById('videoInput')
+
+Promise.all([
+    faceapi.nets.faceRecognitionNet.loadFromUri('/models'),
+    faceapi.nets.faceLandmark68Net.loadFromUri('/models'),
+    faceapi.nets.ssdMobilenetv1.loadFromUri('/models') //heavier/accurate version of tiny face detector
+]).then(start)
+
+function start() {
+    document.body.append('Models Loaded')
+
+    navigator.getUserMedia(
+        { video:{} },
+        stream => video.srcObject = stream,
+        err => console.error(err)
+    )
+
+    //video.src = '../videos/speech.mp4'
+    console.log('video added')
+    recognizeFaces()
+}
+
+async function recognizeFaces() {
+
+    const labeledDescriptors = await loadLabeledImages()
+    //console.log(labeledDescriptors)
+    const faceMatcher = new faceapi.FaceMatcher(labeledDescriptors, 0.7)
+
+
+    video.addEventListener('play', async () => {
+        console.log('Playing')
+        const canvas = faceapi.createCanvasFromMedia(video)
+        document.body.append(canvas)
+
+        const displaySize = { width: video.width, height: video.height }
+        faceapi.matchDimensions(canvas, displaySize)
+
+
+
+        setInterval(async () => {
+            const detections = await faceapi.detectAllFaces(video).withFaceLandmarks().withFaceDescriptors()
+
+            const resizedDetections = faceapi.resizeResults(detections, displaySize)
+
+            canvas.getContext('2d').clearRect(0, 0, canvas.width, canvas.height)
+
+            const results = resizedDetections.map((d) => {
+                return faceMatcher.findBestMatch(d.descriptor)
+            })
+            results.forEach( (result, i) => {
+                const box = resizedDetections[i].detection.box
+                const drawBox = new faceapi.draw.DrawBox(box, { label: result.toString() })
+                console.log(result)
+                drawBox.draw(canvas)
+                setTimeout(() => {
+                    if(result.label == "unknown") {
+                        document.location = "http://localhost:3000/verification/alert-owner"
```

Comment:
The endpoint should be changed.
```suggestion
                        document.location = "http://localhost:3000/verification/owner-verify"
```